lib/rucio/transfertool/fts3.py

```diff
@@ -615,7 +615,7 @@ def _transfer_link(self) -> str:
 
     def _find_attribute_updates(self, request: dict, new_state: RequestState, reason: str, overwrite_corrupted_files: Optional[bool] = None) -> Optional[dict[str, Any]]:
         attributes = None
-        if new_state == RequestState.FAILED and 'Destination file exists and overwrite is not enabled' in (reason or ''):
+        if new_state == RequestState.FAILED and ('Destination file exists and overwrite is not enabled' in (reason or '') or 'Destination file exists and is on tape' in (reason or '')):
```

Comment:
Same as other comment - unsure why we're checking `in (reason or '')` instead of checking for `if reason`